compiler/rustc_codegen_gcc/src/int.rs

```diff
@@ -364,80 +354,85 @@ impl<'a, 'gcc, 'tcx> Builder<'a, 'gcc, 'tcx> {
         (res.dereference(self.location).to_rvalue(), overflow)
     }
 
+    /// Non-`__builtin_*` overflow operations with a `fn(T, T, &mut i32) -> T` signature.
     pub fn operation_with_overflow(
         &self,
         func_name: &str,
         lhs: RValue<'gcc>,
         rhs: RValue<'gcc>,
+        width: u64,
     ) -> (RValue<'gcc>, RValue<'gcc>) {
         let a_type = lhs.get_type();
         let b_type = rhs.get_type();
         debug_assert!(a_type.dyncast_array().is_some());
         debug_assert!(b_type.dyncast_array().is_some());
+        let oflow_type = self.i32_type;
+        let oflow_param_type = oflow_type.make_pointer();
+        let res_type = a_type;
+
+        let oflow_value = self.current_func().new_local(self.location, oflow_type, "overflow");
```

Comment:
Please rename all `oflow` to `overflow` to make it a bit clearer.

Reply:
This is done